gradle.properties

```diff
@@ -0,0 +1,2 @@
+ossrhUsername=dummyId
```

Comment:
travis will Need this file during the build. If we want to skip the checkin we Need to create it with a new build in the .travis.yml or add it to gitignore if we want to avoid later checkins.